docs/grafana.rst

```diff
@@ -0,0 +1,320 @@
+
+*
+
+    The **Iteration View** shows the output of a single iteration within BEAM and plots metrics on an hourly time scale,
+    this is useful for seeing granular dynamics like the use of charging infrastructure over time.
+
+*
+
+    In both of the previous views, only one BEAM run may be viewed at one time but there are also the **Global View Comparison**
+    and **Iteration View Comparison** which allow you to view two BEAM runs at the same time or one BEAM run but with different iterations.
+
+* The **Iteration Map View** allows you to see various BEAM metrics displayed on a map with ability to choose any combination of available metrics, hour and iteration to display.
+
+Select dashboard
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To choose a dashboard to view one can click on a dashboard name:
+
+.. figure:: _static/figs/grafana-dashboard-switch.png
+
+    choose dashboard
+
+Select BEAM run
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To select which BEAM run graphs should be displayed one can use the switch ``run name``. A BEAM run name has the format: *date_time_simulation-name*
+where *date* and *time* are local date and time for pc where BEAM is running and the simulation name is taken from ``beam.agentsim.simulationName``
+config value. The BEAM run list containing the ``run name`` switch is updated after a page refresh. So make sure to refresh your browser after
+a new BEAM run is started and output from the simulation have begun to accumulate. If there are no active BEAM runs in the local history then one will see no choice for ‘run_name’:
+
+.. figure:: _static/figs/grafana-run-name-switch.png
+
+    BEAM run name switch
+
+Disable and enable data traces
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+It is possible to disable and enable some of value traces on a graph by clicking on the name of a trace.
+
+.. figure:: _static/figs/grafana-enable-disable-traces.png
+
+    enable | disable traces
+
+Select BEAM iteration
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+On Iteration View it is possible to select the iteration to see through the ‘iteration_num’ switch. Iteration number switch will update after a page refresh.
+
+.. figure:: _static/figs/grafana-iteration-switch.png
+
+    BEAM iteration switch
+
+Select data update frequency
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+There is a switch to choose how frequent grafana should gather data from the output DB. The switch is in the top right corner in all dashboards.
+In the example below the switch is set up to update every 30 seconds. Also, there is a refresh button near the switch, this button
+does not refresh the entire page, run name or iteration num switch, *only the graphs*.
+
+.. figure:: _static/figs/grafana-update-frequency-switch.png
+
+    shoose graphs update frequency
+
+Iteration Map View
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+On the **Iteration Map View** one may choose which data trace displaying as bars graph **(1)**, which data traces are displaying on a map **(2)**
```

Comment:
On the **Iteration Map View** one may choose which data trace are displayed as bar graphs **(1)**, which data traces are displayed on a map **(2)**